From 718996685c2fd0dd3dfc67a0db48e0d35d11ab52 Mon Sep 17 00:00:00 2001
From: Ruihan Li <lrh2000@pku.edu.cn>
Date: Tue, 24 Sep 2024 17:31:54 +0800
Subject: [PATCH 2/4] Mark `iface::packet` as `pub`

---
 src/iface/mod.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/iface/mod.rs b/src/iface/mod.rs
index 5b028d0..31bd380 100644
--- a/src/iface/mod.rs
+++ b/src/iface/mod.rs
@@ -14,7 +14,7 @@ mod rpl;
 mod socket_meta;
 mod socket_set;
 
-mod packet;
+pub mod packet;
 
 #[cfg(feature = "multicast")]
 pub use self::interface::multicast::MulticastError;
-- 
2.47.0

