From d382672bb9deaae183da1816cba22935d25b8fc0 Mon Sep 17 00:00:00 2001
From: Ruihan Li <lrh2000@pku.edu.cn>
Date: Tue, 24 Sep 2024 17:33:15 +0800
Subject: [PATCH 3/4] Mark `InterfaceInner` fields as `pub`

---
 src/iface/interface/mod.rs | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/iface/interface/mod.rs b/src/iface/interface/mod.rs
index 4570069..e4d0720 100644
--- a/src/iface/interface/mod.rs
+++ b/src/iface/interface/mod.rs
@@ -122,8 +122,8 @@ pub struct Interface {
 /// methods on the `Interface` in this time (since its `device` field is borrowed
 /// exclusively). However, it is still possible to call methods on its `inner` field.
 pub struct InterfaceInner {
-    caps: DeviceCapabilities,
-    now: Instant,
+    pub caps: DeviceCapabilities,
+    pub now: Instant,
     rand: Rand,
 
     #[cfg(any(feature = "medium-ethernet", feature = "medium-ieee802154"))]
-- 
2.47.0

