From bb2182f401d83f280cc9abf4a5a43d435d1e68ee Mon Sep 17 00:00:00 2001
From: Ruihan Li <lrh2000@pku.edu.cn>
Date: Thu, 26 Sep 2024 17:02:02 +0800
Subject: [PATCH 4/4] Mark `InterfaceInner` methods as `pub`

---
 src/iface/interface/mod.rs | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/iface/interface/mod.rs b/src/iface/interface/mod.rs
index e4d0720..a49fc2b 100644
--- a/src/iface/interface/mod.rs
+++ b/src/iface/interface/mod.rs
@@ -933,11 +933,11 @@ impl InterfaceInner {
         }
     }
 
-    fn in_same_network(&self, addr: &IpAddress) -> bool {
+    pub fn in_same_network(&self, addr: &IpAddress) -> bool {
         self.ip_addrs.iter().any(|cidr| cidr.contains_addr(addr))
     }
 
-    fn route(&self, addr: &IpAddress, timestamp: Instant) -> Option<IpAddress> {
+    pub fn route(&self, addr: &IpAddress, timestamp: Instant) -> Option<IpAddress> {
         // Send directly.
         // note: no need to use `self.is_broadcast()` to check for subnet-local broadcast addrs
         //       here because `in_same_network` will already return true.
-- 
2.47.0

